libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -411,10 +413,14 @@ void AP_BattMonitor::check_failsafes(void)
                     break;
             }
 
+#ifndef HAL_NO_GCS
```

Comment:
just use GCS_SEND_TEXT() macro and this is handled for you. We'll slowly convert all of ardupilot to use GCS_SEND_TEXT() instead of gcs().send_text(). It makes for less ifdefs and more readable code

Reply:
done